omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_postgres_validator.rb

```diff
@@ -23,7 +23,7 @@ class PostgresqlPreflightValidator < PreflightValidator
   REQUIRED_VERSION  = Gem::Version.new(9.6)
 
   # supported PG version
-  SUPPORTED_VERSION = Gem::Version.new(13.3)
+  SUPPORTED_VERSION = Gem::Version.new(13.4)
```

Comment:
The supported version can be 13 since 13.x to 13.y would be considered a minor version bump that we should be able to do easily.